doc-src/ch-mountsqfs_desc.rst

```diff
@@ -0,0 +1,52 @@
+Synopsis
+========
+
+::
+
+  $ ch-mountsqfs SQFS MOUNTDIR
+
+Description
+===========
+
+Mounts Charliecloud squashfs :code:`SQFS` in target empty directory :code:`MOUNTDIR`
```

Comment:
Suggest `...on target directory MOUNTDIR`.

Does it have to be empty? Do we check?

Reply:
It does have to be empty. We do check but we don't have to. `squashfuse` will error if the directory is not empty. I can make a note `MOUNTDIR` must be empty.